packages/palette/src/elements/Tooltip/Tooltip.tsx

```diff
@@ -106,37 +68,27 @@ export const Tooltip: React.FC<TooltipProps> = ({
       onMouseOut={deactivate}
       onFocus={activate}
       onBlur={deactivate}
-      position="relative"
-      // TODO: Should avoid making assumptions about display
```

Comment:
Now that I'm looking at this I think I can get rid of all the wrapper divs... Let's see.

Reply:
c9e326159e5c94ed26f4535db4108c2a0771daf4 Removes the need for the wrapper divs. I think this makes more sense since you can then count on this not to mess with positioning, etc of your chosen anchor. The only minor caveat is that the thing you're anchoring to has to accept a ref — this isn't a problem in the vast majority of cases; but I happened to notice that Icons don't forward their refs.